configure.ac

```diff
@@ -285,7 +292,8 @@ RKT_ITERATE_FLAVORS([${RKT_STAGE1_FLAVORS}],[flavor],
                              [coreos],
                                      [RKT_COMMON_COREOS_PROGS],
                              [kvm],
-                                     [AC_MSG_NOTICE([will build linux kernel and lkvm from source, make sure that all their build requirements are fulfilled])
+                                     [AC_MSG_NOTICE([will build linux kernel and ${RKT_STAGE1_KVM_HV} from source, make sure that all their build requirements are fulfilled])
+				      AC_MSG_NOTICE([make sure that hw virtualization is enabled on your system])
```

Comment:
This is being built on the build system, does it require hw-virt? I think you are speaking about the target system, or you'd want to mention running (and/or tests).